AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -159,6 +191,7 @@ - (ASLayout *)filteredNodeLayoutTree
     if (self != context.layout && context.layout.type == ASLayoutElementTypeDisplayNode) {
       ASLayout *layout = [ASLayout layoutWithLayout:context.layout position:context.absolutePosition];
       layout.flattened = YES;
+      layout.cacheSublayoutLayoutElements = YES;
```

Comment:
Nice.  This probably makes sense to keep like this, although we could make it implied by flattened = YES.

Should we call it .retainsSublayoutElements?  It's maybe not as clear as SublayoutLayoutElements, but using Retains instead of Cache and shortening it will probably make it understandable still.